text-generation-inference/server/text_generation_server/jetstream_pt_support/models/mixtral_model_hf.py

```diff
@@ -4,6 +4,20 @@
 from transformers import GenerationConfig, GenerationMixin, MixtralConfig
 
 
+class MixtralConfigHf(MixtralConfig, mixtral_config.ModelArgs):
```

Comment:
This is a bit brittle, as setting one argument will not update its aliased value. What is the benefit of adding this ?

Reply:
The `Transformer` class from Jetstream is the model class for Mistral, and it has a config variable that used to be an instance of `ModelArgs`. When I use the model in TGI and want to use the `GenerationMixin` methods for sampling, it ends up using the config, that it expects it to be a transformers' `MixtralConfig` instance, and it fails because it is not. This is why I came up with this solution. I could try doing composition instead of heritage to see if I can find a cleaner solution though.